Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -700,7 +700,6 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
             }
 
             Logger.networkProtection.error("🔴 Stopping VPN due to no auth token")
-            await cancelTunnel(with: TunnelError.startingTunnelWithoutAuthToken)
```

Comment:
No need to cancel the tunnel if we're throwing an error below, which is the right way to cancel startup.

Reply:
Nice, yeah good catch.